env0/resource_template.go

```diff
@@ -208,6 +211,20 @@ func getTemplateSchema(prefix string) map[string]*schema.Schema {
 			ConflictsWith: allVCSAttributesBut("is_azure_devops", "token_id"),
 			RequiredWith:  requiredWith("token_id"),
 		},
+		"helm_chart_name": {
+			Type:          schema.TypeString,
+			Optional:      true,
+			Description:   "the helm chart name. Required if is_helm_repository is set to 'true'",
+			ConflictsWith: allVCSAttributesBut("helm_chart_name", "is_helm_repository"),
+		},
+		"is_helm_repository": {
+			Type:          schema.TypeBool,
+			Optional:      true,
+			Description:   "true if this template integrates with a helm repository",
+			Default:       "false",
+			ConflictsWith: allVCSAttributesBut("helm_chart_name", "is_helm_repository"),
+			RequiredWith:  requiredWith("helm_chart_name"),
```

Comment:
I'm not really in the details about the `RequiredWith` and `ConflictsWith` logic but overall - one cannot be defined without the other, right?

Reply:
correct.
It's even more than that. For example, the other "is_*_" cannot be set with it. (allVCSAttributesBut).